python/ccxt/deribit.py

```diff
@@ -3071,9 +3071,13 @@ def fetch_funding_rate_history(self, symbol: Str = None, since: Int = None, limi
         market = self.market(symbol)
         paginate = False
         paginate, params = self.handle_option_and_params(params, 'fetchFundingRateHistory', 'paginate')
+        maxEntriesPerRequest = 744  # seems exchange returns max 744 items per request
```

Comment:
generated files are committed

Reply:
yes sure, but in this PR I've specifically pushed the .py files so the user would have seen the change and tested himself (as he was trying to modify files manually to see the working code).